src/lua/log.lua

```diff
@@ -647,24 +773,26 @@ local compat_v16 = {
     end;
 }
 
-local log = {
+log = {
     warn = say_closure(S_WARN),
     info = say_closure(S_INFO),
     verbose = say_closure(S_VERBOSE),
     debug = say_closure(S_DEBUG),
     error = say_closure(S_ERROR),
     rotate = log_rotate,
     pid = log_pid,
-    level = log_level,
-    log_format = log_format,
+    level = set_log_level,
+    log_format = set_log_format,
     cfg = setmetatable(log_cfg, {
-        __call = load_cfg,
+        __call = log_configure,
     }),
     box_api = {
         cfg_get = box_api_cfg_get,
         cfg_set = box_api_cfg_set,
         cfg_set_log_level = box_api_cfg_set_log_level,
         cfg_set_log_format = box_api_set_log_format,
```

Comment:
I found it confusing at first that this patch leaves the old methods be although it looks like they are never used. I now realize that they are removed in the next patch. Why not squash the patches to avoid confusion?